event_routing_backends/utils/xapi_lrs_client.py

```diff
@@ -75,7 +75,7 @@ def bulk_send(self, statement_data):
         response = self.lrs_client.save_statements(statement_data)
 
         if not response.success:
-            if response.response.code == 409:
+            if response.response.code == 409 or response.response.code == 204:
```

Comment:
You could just do `in (204, 409)` here

Reply:
This one is never reached as the json decode error is triggered first. I will remove it